src/updater.js

```diff
@@ -57,7 +57,8 @@ const offlineUpdater = function offlineUpdater(
   }
 
   if (action.type === PERSIST_REHYDRATE) {
-    return { ...state, busy: false };
+    if (action.payload.offline) delete action.payload.offline.online;
+    return { ...state, ...action.payload.offline, busy: false };
```

Comment:
Rather than deleting from the payload it might be simpler and involve less mutation if you just do:

```
return { ...state, ...action.payload.offline, online: state.online, busy: false };
```